test/parallel/test-fs-stat-bigint.js

```diff
@@ -66,16 +66,16 @@ function verifyStats(bigintStats, numStats) {
       const msFromBigIntNs = Number(nsFromBigInt / (10n ** 6n));
       const msFromNum = numStats[key];
       // The difference between the millisecond-precision values should be
-      // smaller than 2
+      // smaller than 5 - i9 consistently returns 4ms diff
```

Comment:
Nit: I don't think this will help clarify for `5` magic number so probably not needed.
```suggestion
      // smaller than 5
```

Reply:
```suggestion
      // It's possible that the file stats were updated between the two
      // statSync() calls so allow for a small (5ms) difference.
```